test/image/mocks/zzz_shape_shift_horizontal.json

```diff
@@ -0,0 +1,87 @@
+{
+    "data": [
+        {
+            "y": [
+                ["A", "A", "B", "B"], ["C", "D", "C", "D"]
+            ],
+            "x": [
+                1, 2, 3, 4
+            ],
+            "type": "bar",
+            "marker": {
+                "color": "rgba(153, 217, 234, 1)"
+            },
+            "orientation": "h"
+        },
+        {
+            "y": [
+                ["A", "A", "B", "B"], ["C", "D", "C", "D"]
+            ],
+            "x": [
+                4, 3, 2, 1
+            ],
+            "type": "bar",
+            "orientation": "h"
+        }
+    ],
+    "layout": {
+        "shapes": [
+            {
+                "layer": "above",
+                "type": "rect",
+                "label": {
+                    "text": "around A,D",
+                    "textposition": "bottom center",
+                    "textangle": 0
+                },
+                "line": {
+                  "color": "black",
+                  "width": 3.0
+                },
+                "y0": ["A", "C"],
```

Comment:
Yeah to know that this surrounds A,D you have to know that C is right before D, which is fragile. Much better in this case is if we had a way to specify two shifts per dimension.